### PR TITLE
Choose payment method during premium account setup #4741 - PAD UI only

### PR DESCRIPTION
**Pull request by nitheesh-aot**
*Issue #:*
https://github.com/bcgov/entity/issues/4741

*Description of changes:*
- Choose payment method during premium account setup - PAD UI only

*Checklist:*
I confirm that below checklist items are addressed in this pull request; (check the boxes applicable)
- [ ] No lint errors
- [ ] No test case failures and proper coverage for all modules/classes
- [ ] Updated the deployment configs for new environment variable(s)
- [ ] Updtaed the postman collection in entity repository (https://github.com/bcgov/entity/tree/master/api-e2e/postman) for e2e tests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

**Comment by nitheesh-aot** [comment]
only PAD ui of the ticket, more changes in next PR.

**Review by mengdong19 — APPROVED** [review]

File: auth-web/src/components/auth/common/PaymentMethods.vue [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+@Component({
+  components: {
+    PADInfoForm
+  }
+})
+export default class PaymentMethodSelector extends Vue {
+  @Prop({ default: '' }) currentOrgType: string
+  private selectedPaymentMethod: string = ''
+  private paymentTypes = PaymentTypes
+  private paymentsPerAccountType = {
```
Comment (saravanpa-aot) [inline auth-web/src/components/auth/common/PaymentMethods.vue:114]: can it be in a more common place?if it makes sense

File: auth-web/src/components/auth/common/PaymentMethods.vue [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+    </v-card>
+  </div>
+</template>
+
+<script lang="ts">
+import { Account, PaymentTypes } from '@/util/constants'
+import { Component, Emit, Mixins, Prop, Vue } from 'vue-property-decorator'
+import PADInfoForm from '@/components/auth/common/PADInfoForm.vue'
+
+const PAYMENT_METHODS = {
```
Comment (saravanpa-aot) [inline auth-web/src/components/auth/common/PaymentMethods.vue:64]: same for this..may be another common place

**Review by saravanpa-aot — APPROVED** [review]

File: auth-web/src/components/auth/common/PaymentMethods.vue (hunk 2 above)
Comment (nitheesh-aot) [reply auth-web/src/components/auth/common/PaymentMethods.vue:64]: the scope of this only applies to this component now. and I am planning to reuse the same component in the account settings as well. will disucss.

File: auth-web/src/components/auth/common/PaymentMethods.vue (hunk 1 above)
Comment (nitheesh-aot) [reply auth-web/src/components/auth/common/PaymentMethods.vue:114]: Yes, this I think we can move to either constants or config-helper file.

**Pull request merged**